src/node_file.cc

```diff
@@ -2727,6 +2730,270 @@ static void Mkdtemp(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+bool FileURLToPathWin32Impl(Environment* env,
```

Comment:
Please initialize a single FileURLToPath function for all environments. No need to name this function Win32.